tardis/plugins/prometheusmonitoring.py

```diff
@@ -0,0 +1,75 @@
+from ..configuration.configuration import Configuration
+from ..interfaces.plugin import Plugin
+from ..interfaces.state import State
+from ..interfaces.siteadapter import ResourceStatus
+from ..utilities.attributedict import AttributeDict
+
+import logging
+from aioprometheus import Service, Gauge
+
+
+class PrometheusMonitoring(Plugin):
+    """
+    The :py:class:`~tardis.plugins.prometheusmonitoring.PrometheusMonitoring`
+    implements an interface to monitor the state of the Drones using Prometheus.
+    """
+
+    def __init__(self):
+        self.logger = logging.getLogger("prometheusmonitoring")
```

Comment:
@giffels Should we enforce cobald channels here already (see issue #137)?

Reply:
@maxfischer2781 Agreed!

Reply:
Something like this?

```python
self.logger = logging.getLogger("cobald.monitor.prometheusmonitoring")
```

Reply:
I think in order to get this merged, we will start with

```python
self.logger = logging.getLogger("cobald.runtime.tardis.plugins.prometheusmonitoring")
```

and can adjust it depending on the discussion ongoing in #137 .